systems/plants/@RigidBodyManipulator/doKinematics.m

```diff
@@ -421,11 +422,18 @@
 end
 
 function qd = computeQdot(bodies, vToqdot, v, nq)
-qd = zeros(nq, 1) * vToqdot{2}(1);
-for i = 2 : length(bodies)
-  body = bodies(i);
-  qd(body.position_num) = vToqdot{i} * v(body.velocity_num);
+original_warning_state = warning('error', 'Drake:TaylorVar:DoubleConversion');
```

Comment:
ah, found it here:  http://undocumentedmatlab.com/blog/trapping-warnings-efficiently
can you add a comment to this line saying that it an undocumented way to turn the specified warning ID into an error.